ThunderCloud/Quiz.swift

```diff
@@ -8,6 +8,36 @@
 
 import UIKit
 
+// MARK: - QuizConfiguration
+
+/// App level customization of `Quiz`
+public struct QuizConfiguration {
+    
+    /// Shared `QuizConfiguration` singleton
+    public static var shared = QuizConfiguration()
+    
+    /// Each time a user retakes the test, questions appear in different order
+    public var shuffleQuestions = false
+    
+    /// If `true`, the `QuizQuestionViewController` `continueButton` will
+    /// enable/disable itself depending on whether the question has been answered.
+    public var requireAnswer: Bool = true
+    
+    /// Default init
+    init() {
+    }
+    
+    /// Default public memeberwise `init`
+    /// - Parameters:
+    ///   - shuffleQuestions: Whether quiz questions should be shuffled
+    ///   - requireAnswer: Whether answers are required before progressing to next question
+    public init (shuffleQuestions: Bool = false, requireAnswer: Bool = true) {
+        self.shuffleQuestions = shuffleQuestions
```

Comment:
@simonmitchell `requireAnswer` is not being set

Reply:
Good catch!

Reply:
Has this been pushed?